documentation/Functions.md

```diff
@@ -35,13 +35,21 @@ installDevonfwIde(["java","mvn"], "2020.08.001")
 ***
 
 ### restoreDevonfwIde <a name="restoreDevonfwIde"></a>
+It is similar to [installDevonfwIde](#installDevonfwIde) function. With the installDevonfwIDE function a script will load a settings file for the installation to contains the tools you have chosen. With this function, the whole installation of the devonfwIde will be done by a script in the background and the Katakoda user won't have to download a start the installation.
```

Comment:
The second `With the installDevonfwIde` is misspelled.
Use `devonfw-ide` instead of `devonfwIde`

Reply:
I think we do not need to mention what the function does (load settings, etc)
Just say that it is similar to installDevonfwIde, that it has the same parameters and that the difference between the two functions is that restoreDevonfwIde installs the ide through a script in the background and the user does not have to download and start the installation.